test/ex_force_test.exs

```diff
@@ -286,6 +286,64 @@ defmodule ExForceTest do
            }
   end
 
+  test "create_sobject - success", %{bypass: bypass} do
+    Bypass.expect_once(bypass, "POST", "/services/data/v40.0/sobjects/Account/", fn conn ->
+      {:ok, raw, conn} = Conn.read_body(conn)
+      expected_body = %{"FirstName" => "first_name"}
+      ^expected_body = Poison.decode!(raw)
+      resp_body = """
+      {
+        "id": "001D000000IqhSLIAZ",
+        "errors": [],
+        "success": "true"
```

Comment:
When I tested it manually for v40.0, it returns `warnings` as well. Could you check that out? I'd like to keep it close to real response as much as possible.

Reply:
hum. I ll try it again but did not see any warnings. let me check  But that is what they document at https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_sobject_create.htm

Reply:
you are right, I see the warnings field too. I ll fix  that but poor documentation on salesforce end. Cannot see it in their Rest doc...